stl/inc/condition_variable

```diff
@@ -156,43 +174,19 @@ private:
         return reinterpret_cast<_Cnd_t>(&_Cnd_storage);
     }
 
+    template <class _Lock>
+    void _Unlock_relock(_Lock& _Lck) noexcept /* terminates */ {
+        _Lck.unlock(); // could throw
+        _Lck.lock();

```

Comment:
According to WG21-N4861 32.5.3.2 [[thread.mutex.requirements.mutex]/26](https://eel.is/c++draft/thread.mutex.requirements.mutex#26), `unlock()` can't throw, but `lock()` can throw ([/12](https://eel.is/c++draft/thread.mutex.requirements.mutex#12)).